ui-tests/src/main/java/org/jenkinsci/test/acceptance/plugins/warnings_ng/AbstractIssuesTableRow.java

```diff
@@ -0,0 +1,23 @@
+package org.jenkinsci.test.acceptance.plugins.warnings_ng;
+
+/**
+ * Abstract representation of a table row of the issues-table.
+ *
+ * @author Stephan Plöderl
+ */
+public abstract class AbstractIssuesTableRow {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [This abstract class does not have any abstract methods](https://app.codacy.com/manual/uhafner/warnings-ng-plugin/pullRequest?prid=5070362)